sympy/stats/symbolic_probability.py

```diff
@@ -173,6 +229,8 @@ def expand(self, **hints):
                 else:
                     nonrv.append(a)
             return Mul(*nonrv)*Expectation(Mul(*rv), condition=condition)
+        elif isinstance(_expand(expr), Add):
+            return Expectation(_expand(expr)).expand()
```

Comment:
Calling `_expand(expr)` twice? Furthermore, `.expand()` is expensive... you know you just need to distribute an addition over `Expectation`, so...
```python
r = _expand(expr)
return Add.fromiter(Expectation(i) for i in r)
```